vm/src/stdlib/os.rs

```diff
@@ -151,6 +151,48 @@ fn os_remove(path: PyStringRef, vm: &VirtualMachine) -> PyResult {
     Ok(vm.get_none())
 }
 
+fn os_mkdir(path: PyStringRef, vm: &VirtualMachine) -> PyResult {
+    match fs::create_dir(&path.value) {
+        Ok(_) => (),
+        Err(s) => return Err(vm.new_os_error(s.to_string())),
+    }
+
+    Ok(vm.get_none())
```

Comment:
With the new args style, we don't need to explicitly return `None` anymore. If you change the return type to `PyResult<()>` you can return `Ok(())`. Also I'd just do this directly in the `match` above so that the early return is not necessary.

Reply:
You could also just use `map_err` instead of the `match`.

Reply:
Thanks @OddCoincidence and @coolreader18. It is way simpler this way :)